### ISSUE TITLE
Annotate axioms in output ontology with prior probabilities

### ISSUE THREAD

**Issue by cmungall** [INCATools/boomer#147]
E.g. if the txt output is

```
A:1 ProperSubClassOf B:1        (most probable) 0.7
A:2 ProperSuperClassOf B:2      (most probable) 0.7
A:3 EquivalentTo B:3    (most probable) 0.7
A:4 SiblingOf B:4       (most probable) 0.7
```

then add annotations to axioms 1-3. TBD: predicate for probability? biollink?

Also:

currently no axiom is emitted for 4. Can we emit an annotation assertion? It is useful to be explicit about what was not inferred

Also:

Annotations on ontology:

 - ontology ID. Get from user?
 - metadata about the input ontologies, parameters, steps. Consider using a full PROV model. But even just some rdfs:comments would be awesome

**Comment by balhoff** [comment]
I don't think it suits your use case, but just FYI you can output axioms for 4 (and the "proper" aspect of 1 and 2) by using an undocumented option `--output-internal-axioms=true`. This includes the generated disjoint siblings.

**Comment by cmungall** [comment]
Thanks, I can see that being useful for doing things like debugging and generating explanations. But yes just a direct annotation axiom is what is best here
